### ISSUE TITLE
"expression search-NG" blocks general search Ctrl-K

### ISSUE THREAD

**Issue by GoeWink** [opto/Expression-Search-NG#89]
If a search is started with "Ctrl-K" (over all folders) and results are shown with a filter (I never used it), I click "all results as a list" It results in an empty window with the message "Loading". The same thing if you double click on your individual message. Where does this come from and above all: how to change it?!
This happens after the last Update in TB115 (Betterbird)

**Comment by atf2** [comment]
Just to confirm that I see this too, except that I don't even see a message "Loading", just a blank tab with a folder icon but no name in the tab header.  I am using Thunderbird 115.4.2 with Windows 11 v10.0.22621 (64 bit).  This broken behaviour occurs if the "Expression Search - NG" extension is enabled, but not if it is simply installed but disabled.  However, the inability to see the results of my global searches rather undermines the value of the extension.

**Comment by opto** [comment]
please try this version. ctrl k works here on winn 11, 64 bit, TB 115.4.2

https://github.com/opto/Expression-Search-NG/releases/tag/v4.0.13

**Comment by opto** [comment]
doublicate of #80?

**Comment by TriassicExtinction** [comment]
> doublicate of #80?

Version 4.0.13 now allows QuickFilter to work inside a global search tab.  However, Expression Search does not actually search, e.g. "f:<name>" does not search for <name>.  The normal QuickSearch filter works however.

**Comment by oehm-smith** [comment]
Yes that fixed my problem.  Well done opto for getting this out.

**Comment by opto** [comment]
@TriassicExtinction your observation is different from #80.

As it is a new issue, I added it as #99, whereas this one is resolved
